gi/object.cpp

```diff
@@ -2444,6 +2437,85 @@ gjs_object_set_gproperty (GObject      *object,
     g_free (underscore_name);
 }
 
+static JSBool
+gjs_override_property(JSContext *cx,
+                      unsigned   argc,
+                      jsval     *vp)
+{
+    jsval *argv = JS_ARGV(cx, vp);
```

Comment:
I don't think I can do that without changing the API of `gjs_parse_args`, which in my opinion would prohibitively inflate this patch.